common/components/common/projects/EditProjectForm.jsx

```diff
@@ -9,8 +9,10 @@ import FileUploadList from '../../../components/forms/FileUploadList.jsx'
 import TagCategory from '../../common/tags/TagCategory.jsx'
 import TagSelector from '../tags/TagSelector.jsx'
 import DjangoCSRFToken from 'django-react-csrftoken'
+import FormValidation from '../../../components/forms/FormValidation.jsx'
```

Comment:
higher level comment, out of scope of this diff: it'd be dope if you could find a way that these import statements didn't have to be path dependent so you could just do
```
import FormValidation from 'FormValidation.jsx'
```

Reply:
Yeah, dealing is with relative paths is annoying.  Created a backlog item for it here: https://trello.com/c/lxulvsol